### PR TITLE
Readme update

### LINKED ISSUE #372: Document how to use this repo
We don't actually have that many dependencies.  Maybe we change our install instructions to just have them install `astrodbkit2` and maybe also `ads`.

Maybe we want to do the packaging work to make `pip install simple`.

### LINKED ISSUE #459: Update README for general users
Right now, the README is focused on new developers and using the Python interface.  

- [x] Add a section for folks who may just want to use the .sqlite file.  
- [x] fix schema figure is currently broken. #364 
- [ ] add bit about `astrodb_utils` package for ingesting. 
- [x] maybe add  something about tests? 
- [x] demonstrate the `load_astrodb` function.

### PR DESCRIPTION
**Pull request by kelle**
Updates Getting Started instructions and adds a Contributor Instructions section. Also fixes schema image.

Closes #372 
Closes #459

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,14 @@ imaged exoplanets: a simple archive of complex objects. The tables and fields cu
...
-While we are using brown dwarfs to build out the SIMPLE database, our intention is to build a database schema and 
-software which could be used by other subfields to roll their own collaborative databases and web interfaces.
-
-This database uses the [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/14/orm/index.html) and is designed to be
-interacted with via the `astrodbkit2` package.
-
-If you'd like to participate or just stay in the loop as this project progresses, please request to join this discussion
- list:
-https://groups.google.com/forum/#!forum/simple-archive
+To report missing data, data bugs or make feature requests, please use the [SIMPLE-db issue tracker](https://github.com/SIMPLE-AstroDB/SIMPLE-db/issues).
```
Comment (Will-Cooper) [inline README.md:15]: do we want to point directly to some issue templates as well?

File: README.md [hunk 2]
```diff
@@ -31,37 +25,28 @@ If you'd like to set up your own copy of the SIMPLE database, here's what we rec
 
 1. Clone or download a copy of this repo locally onto your computer.
  
-2. Set up an environment for the python code. 
+2. Set up an environment for the Python code and install dependencies.
```
Comment (Will-Cooper) [inline README.md:28]: could we also give some instructions for just using pip?

**Review by Will-Cooper — APPROVED** [review]
I like it!

File: README.md (hunk 2 above)
Comment (dr-rodriguez) [reply README.md:28]: I think it's more important that we have pip instructions than conda ones; some environments (like Codespaces) do not come with conda.

File: README.md [hunk 3]
```diff
@@ -73,12 +58,28 @@ For example:
         ```
         db.inventory('2MASS J01415823-4633574', pretty_print=True)
         ```
+5. The database can also be modified using helper scripts found in [`simple/utils`](simple/utils) and in the `astrodb_utils` package. Previously used scripts to modify and/or update the database are stored in the [`scripts/`](scripts) directory and can be used for inspiration.
+
+## Contributor Instructions
+If you've made changes to the SIMPLE Archive that you would like to contribute to the public verion, here's how to make a contribution.
+
+1. If you've made changes using Python or the DB Browser, write out the new/modified JSON files:
```
Comment (dr-rodriguez) [inline README.md:66]: Suggested rewrite:
`If you've made changes to the database contents, write out the new/modified JSON files with:`
(no need to specify python/DB browser)

File: README.md [hunk 4]
```diff
@@ -31,37 +25,28 @@ If you'd like to set up your own copy of the SIMPLE database, here's what we rec
...
-   ```
-5. Load in the database by reading in the directory where the JSON files are located
+   db = load_astrodb("SIMPLE.sqlite", recreatedb=True,  reference_tables=REFERENCE_TABLES)
+    ```
 
-   ```python
-   db.load_database('data/')
-   ```
-6. Use `astrodbkit2` to [explore](https://astrodbkit2.readthedocs.io/en/latest/#exploring-the-schema), [query](https://astrodbkit2.readthedocs.io/en/latest/#querying-the-database), and/or [modify](https://astrodbkit2.readthedocs.io/en/latest/#modifying-data) the database.
+    This step generates a "SIMPLE.sqlite" file which can be opened, explored, and modified using the [DB Browser for SQLite](https://sqlitebrowser.org/).
```
Comment (dr-rodriguez) [inline README.md:47]: Suggested rewrite:
`and modified using a variety of tools`
I know you love DB Browser, but I think the mention of it around line 13 is sufficient. I don't want to give the impression that this is the only/recommended way of using the database, **especially** for modifications.

File: README.md [hunk 5]
```diff
@@ -73,12 +58,28 @@ For example:
...
+    ```python
+    db.save_database(directory="data/")
+    ```
+
+2. (Optional) Run and modify tests as necessary
+    ```bash
+    pytest
+    ```
+
+3. Open a pull request with the modified JSON files and optionally, your ingest script.
```
Comment (dr-rodriguez) [inline README.md:76]: optionally? I thought we required the ingest scripts?

File: README.md (hunk 2 above)
Comment (kelle) [reply README.md:28]: If someone wants to provide me with pip instructions, I'm happy to include them.  Maybe opening a dedicated  issue is the way to go here.

File: README.md (hunk 1 above)
Comment (kelle) [reply README.md:15]: Not yet, most of the templates use old methods.

Comment (kelle) [reply README.md:15]: When they click New issue, they'll get an option for the templates. I don't think it's necessary here.

File: README.md (hunk 2 above)
Comment (dr-rodriguez) [reply README.md:28]: Created https://github.com/SIMPLE-AstroDB/SIMPLE-db/issues/493, I think we should handle that outside this PR. Looking at how the tests run is a good starting location, but there are ways we can simplify that for the user.

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**